pkgs/development/compilers/elm/packages/avh4-lib.nix

```diff
@@ -0,0 +1,29 @@
+{ mkDerivation, ansi-terminal, ansi-wl-pprint, array, base, bimap
+, binary, bytestring, containers, directory, fetchgit, filepath
+, lib, mtl, process, relude, tasty, tasty-discover, tasty-hspec
+, tasty-hunit, text
+}:
+mkDerivation {
+  pname = "avh4-lib";
+  version = "0.0.0.1";
+  src = fetchgit {
+    url = "https://github.com/avh4/elm-format";
+    sha256 = "0bcjkcs1dy1csz0mpk7d4b5wf93fsj9p86x8fp42mb0pipdd0bh6";
+    rev = "80f15d85ee71e1663c9b53903f2b5b2aa444a3be";
+    fetchSubmodules = true;
+  };
+  postUnpack = "sourceRoot+=/avh4-lib; echo source root reset to $sourceRoot";
```

Comment:
What is this doing?

Reply:
This is building a haskell library that's in a folder within the git repo.  This whole file was generated via `cabal2nix --no-haddock https://github.com/avh4/elm-format --revision "0.8.5" --subpath avh4-lib`

Reply:
Ah thanks. I see that repository is now sort of monorepo containing few individual libraries :+1: Makes sense.